certified-connectors/Cognizant Automation Center/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1105 @@
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/portal/ino/api/v3/collab/activity/{ActivityID}/actions/{ActivityActionID}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Success": {
+                  "type": "boolean",
+                  "description": "Success"
+                },
+                "DateTZConversionColumn": {
+                  "type": "array",
+                  "items": {},
+                  "description": "DateTZConversionColumn"
+                },
+                "Data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "Status": {
+                        "type": "string",
+                        "description": "Status"
+                      },
+                      "Automation Status": {
+                        "type": "string",
+                        "description": "Automation Status"
+                      },
+                      "Status Id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Status Id"
+                      },
+                      "Automation Status Id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Automation Status Id"
+                      },
+                      "Output Status Id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "Output Status Id"
+                      }
+                    }
+                  },
+                  "description": "Data"
+                },
+                "TotalCount": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "TotalCount"
+                },
+                "ColumnMapping": {
+                  "type": "array",
+                  "items": {},
+                  "description": "ColumnMapping"
+                },
+                "DateColumn": {
+                  "type": "array",
+                  "items": {},
+                  "description": "DateColumn"
+                },
+                "ComboBox": {
+                  "type": "array",
+                  "items": {},
+                  "description": "ComboBox"
+                },
+                "ComboSelect": {
+                  "type": "array",
+                  "items": {},
+                  "description": "ComboSelect"
+                },
+                "TransactionKey": {
+                  "type": "string",
+                  "description": "TransactionKey"
+                }
+              }
+            }
+          },
+          "default": {

```

Comment:
@SelvaGanesh-Cognizant Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

Reply:
@sriyen-msft default response is removed. Please check.